DependencyInjection/DoctrineExtension.php

```diff
@@ -800,7 +800,7 @@ protected function loadOrmCacheDrivers(array $entityManager, ContainerBuilder $c
     private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder $container)
     {
         $propertyExtractorDefinition = $container->register(sprintf('doctrine.orm.%s_entity_manager.property_info_extractor', $entityManagerName), DoctrineExtractor::class);
-        if (interface_exists(PropertyInitializableExtractorInterface::class)) {
+        if (is_a(DoctrineType::class, ResetInterface::class, true)) {
```

Comment:
This will fail when the Form component is not installed and thus the parent class of `DoctrineType` does not exist.

Reply:
Looks like this then:

```
$ bin/console cache:clear

 // Clearing the cache for the dev environment with debug true                                                          

PHP Fatal error:  Class 'Symfony\Component\Form\AbstractType' not found in /Users/christian/projects/doctrine-bundle-917/vendor/symfony/doctrine-bridge/Form/Type/DoctrineType.php on line 29
```

Reply:
ah hence the build failure… and I'm afraid this means my solution can't work. Closing.